client/src/components/showMeMore/Download.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import Button from '@material-ui/core/Button';
+import ReactExport from 'react-export-excel';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import buttonsCss from '../../css/buttons';
+
+const { ExcelFile } = ReactExport;
+const { ExcelSheet } = ReactExport.ExcelFile;
+
+// Pre-defined dataset for export
+const dataSet = [
+  {
+    columns: [
```

Comment:
`chestPainSeverity` is no longer used

Missing physical: `giIssueSeverity`, `headacheSeverity`, `lostSmellSeverity`, `lostTasteSeverity`

Do we want to include emotional stuff? `appetite`, `energy`, `interest`, `sadness`, `sleep`?

Ideally, we have all of these fields stored in a constants file or something for less hard coding

Reply:
Added emotional stuff and moved `dataSet` into constants